### PR TITLE
Making parcoords resize

### PR DESCRIPTION
**Pull request by erzel**
Making the parallel-coordinates visualization scale to fit the available space.
@stephanwlee, please take a look.
Thanks!

### CODE REVIEW

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html [hunk 1]
```diff
@@ -104,14 +104,14 @@
...
-        background: black;
-        border-radius: 5px;
-        border: 2px solid black;
-        padding: 8px;
-        box-shadow: 2px 2px #777;
-        max-width: 100%;
-        max-height: 100%;
+      #container {
+        width: 100%;
+        height: 100%;
```
Comment (stephanwlee) [inline tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: nit: sort a-z

Comment (erzel) [reply tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: Done. Though, I don't think I'm doing this in other places. Is there a style-guideline for sorting members of a CSS group somewhere?

Comment (stephanwlee) [reply tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: Ah sorry, I should have linked a rule here. This is the rule: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html [hunk 2]
```diff
@@ -291,13 +275,31 @@
...
-              width = this.width - margin.left - margin.right,
-              height = this.height - margin.top - margin.bottom;
+        // We use the following algorithm for laying out our SVG:
+        // We compute a minimum size for the SVG based on the number columns
+        // and some margins. We set the svg "width" and "height" styles
+        // to "100%" so that it takes up the full area of its parent, but use
+        // "min-width" and "min-height", so that if the parent is too small
+        // the svg won't shrink down (it will overflow with scroll bars).
+        // If the parent is larger than the minimum size, we use the its viewBox
+        // attr to scale the contents to fit the larger size.
```
Comment (stephanwlee) [inline tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: If this was the intent, we can make the viewBox and lay out invariable to the size of the container and just use width/height CSS property on SVG to scale accordingly, right?

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html (hunk 1 above)
Comment (wchargin) [reply tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

or go/css-style internally.

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html (hunk 2 above)
Comment (erzel) [reply tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: I'm not sure I follow. Can you provide some code for your suggestion in a follow-up comment?

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<!--
+r<!--
```
Comment (stephanwlee) [inline tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: extraneous character here.

Comment (erzel) [reply tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: Done.

**Comment by erzel** [comment]
Thanks Stephan!

**Pull request merged**